### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by Uplink036** [Uplink036/StardewCalculator#3]
Is testing important to this project? I genuinely don't know, I kind of want it, but am so new to C# that I don't know what is good. Should we do our own minimal unit test or find a framework? If so, which? Here is a discussion for it. This should be closed and finished before any serious working on the project starts, but isn't an immediate issue.

**Comment by s02blom** [comment]
I mentioned that [Dylan Beattie](https://github.com/dylanbeattie) used something in his project [Starship](https://github.com/dylanbeattie/Starship) to test. Well I looked it up, he's using [NCrunch](https://www.ncrunch.net/) which is not  a testing framework. What it does is continually run your tests during development so that you can get continuous feedback. Starship does not use a testing framework as it is a compiler so Dylan has set up his own testing.

**Comment by s02blom** [comment]
Following [this](https://learn.microsoft.com/en-us/visualstudio/test/walkthrough-creating-and-running-unit-tests-for-managed-code?view=vs-2022) guide I got to [Create a unit test project](https://learn.microsoft.com/en-us/visualstudio/test/walkthrough-creating-and-running-unit-tests-for-managed-code?view=vs-2022) where upon saw that there were 3 different test frameworks. At that point I found [this](https://www.lambdatest.com/blog/nunit-vs-xunit-vs-mstest/) article explaining the differences between them. For us I don't think it matters much as we really aren't familiar with any. I felt most drawn to Nunit, because of the language and the ignore("reason"). xUnit sounds more mature
